### ISSUE TITLE
Add missing documentation to `googlesitekit.modules` functions

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#2414]
Functions that are exposed on `googlesitekit.modules` (wrapping datastore actions) should be documented.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The three functions exposed on `googlesitekit.modules` which wrap datastore action calls should be updated as follows:
    * They should not pass through `...args`, but the actual list of parameters to make the code more clear.
    * They should receive documentation, which essentially should be the same documentation as the respective datastore actions have.
* In other words, similarly to how it already is for the functions on `googlesitekit.widgets`.

## Implementation Brief

* 

### Test Coverage

* 

### Visual Regression Changes

* 

## QA Brief

* 

## Changelog entry

*

**Comment by felixarntz** [comment]
Duplicate of #2272.
